app/src/main/java/com/hamdan/forzenbook/data/database/LoginEntity.kt

```diff
@@ -5,9 +5,11 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 import com.hamdan.forzenbook.data.database.LoginEntity.Companion.TABLE_NAME
 
+// TODO when creating homepage login with token, may need to remove the email portion from this as only a token may be needed
 @Entity(tableName = TABLE_NAME)
 data class LoginEntity(
-    @ColumnInfo(name = TOKEN) val token: String?
+    @ColumnInfo(name = TOKEN) val token: String?,
```

Comment:
You don't need to make these nullable in your entity

Reply:
My mistake, I was thinking that if we could possibly get null from the db we should probably give it nullables as well, but thats just flawed thinking.